bundles/CoreBundle/Resources/translations/en.json

```diff
@@ -992,7 +992,8 @@
     "asset_upload_want_to_overwrite": "Do you want to overwrite file %s?",
     "asset_upload_overwrite": "Overwrite",
     "asset_upload_keep_both": "Keep both files",
-    "asset_upload_overwrite_all": "Overwrite all",
+    "asset_upload_apply_to_all": "Apply to all",
+    "asset_upload_skip": "Skip",
```

Comment:
could you please move these translations to CoreBundle/translations/en.json since CoreBundle/Resources/translations/en.json will be removed soon. 

- thanks

Reply:
@Corepex Done.